pkg/diagnostics/http_monitoring.go

```diff
@@ -230,13 +230,13 @@ func (h *httpMetrics) HTTPMiddleware(next http.HandlerFunc) http.HandlerFunc {
 
 		start := time.Now()
 
-		next(w, r)
+		next.ServeHTTP(w, r)
```

Comment:
Is this the standard way to call next with Chi router?

Reply:
Chi is just a router, this is the standard way of creating a middleware with the std library. https://drstearns.github.io/tutorials/gomiddleware/